test/ConsensusApp/validTransition.test.ts

```diff
@@ -3,134 +3,107 @@ import {ethers} from 'ethers';
 interface ConsensusAppVariablePart {
   appData: string;
   outcome: string;
 }
 
-// constant values
-const emptyOutcome = '0x';
-const defaultOutcome1 = '0x1';
-const defaultOutcome2 = '0x2';
-const defaultOutcome3 = '0x3';
+function constructConsensusVariablePart(
+  furtherVotesRequired: number,
+  currentOutcome,
+  proposedOutcome,
+): ConsensusAppVariablePart {
+  const appData: string = defaultAbiCoder.encode(
+    ['tuple(uint32, bytes)'],
+    [[furtherVotesRequired, proposedOutcome]],
+  );
+  return {appData, outcome: currentOutcome};
+}
 
-const defaultTurn = 1;
+const turnNumB = 1; // not checked by contract
+const numParticipants = 3;
 
-const noValidTransitionError = 'ConsensusApp: No valid transition found for commitments';
-const outcomeMustBeSameError = "ConsensusApp: 'outcome' must be the same between commitments.";
+const noValidTransitionError = 'ConsensusApp: No valid transition found';
 
 beforeAll(async () => {
   consensusApp = await setupContracts(provider, ConsensusAppArtifact);
 });
 
-describe('validTransition', () => {
-  it('valid consensus -> propose', async () => {
-    const variablePartOld = constructConsensusVariablePart(0, defaultOutcome1, emptyOutcome);
-    const variablePartNew = constructConsensusVariablePart(2);
-    const isValid = await consensusApp.validTransition(
-      variablePartOld,
-      variablePartNew,
-      defaultTurn,
-      3,
-    );
-    expect(isValid).toBe(true);
-  });
-
-  it('invalid consensus -> propose: furtherVotesRequired too low', async () => {
-    const variablePartOld = constructConsensusVariablePart(0, defaultOutcome1, emptyOutcome);
-    const variablePartNew = constructConsensusVariablePart(1, defaultOutcome1, defaultOutcome1);
-
-    await expectRevert(
-      () => consensusApp.validTransition(variablePartOld, variablePartNew, defaultTurn, 3),
-      noValidTransitionError,
-    );
-  });
-
-  it('valid vote', async () => {
-    const variablePartOld = constructConsensusVariablePart(2);
-    const variablePartNew = constructConsensusVariablePart(1);
-    expect(
-      await consensusApp.validTransition(variablePartOld, variablePartNew, defaultTurn, 3),
-    ).toBe(true);
-  });
-
-  it('invalid vote: furtherVotesRequired not decreased', async () => {
-    const variablePartOld = constructConsensusVariablePart(2);
-    const variablePartNew = constructConsensusVariablePart(2);
-
-    await expectRevert(
-      () => consensusApp.validTransition(variablePartOld, variablePartNew, defaultTurn, 4),
-      noValidTransitionError,
-    );
-  });
-
-  it('valid veto', async () => {
-    const variablePartOld = constructConsensusVariablePart(2, defaultOutcome1, defaultOutcome2);
-    const variablePartNew = constructConsensusVariablePart(0, defaultOutcome1, emptyOutcome);
-    expect(
-      await consensusApp.validTransition(variablePartOld, variablePartNew, defaultTurn, 3),
-    ).toBe(true);
-  });
+const description00 = 'valid consensus -> propose';
```

Comment:
Nit, what's the downside of just using `description0`, `description1`, etc constant names? Prettier seems to format the table where these constants are used nicely even when one of the description constants (`description10` in this case) has a different length than the others.

Reply:
Yeah you're right the table ends up looking nice. But the lists of constants itself looks less nice... 

This may have been a step too far :-P

I'm anticipating that we might be able to do away with the `description0` constants in near-future PRs.